Mixer.Base/Clients/MixerWebSocketClientBase.cs

```diff
@@ -183,6 +183,10 @@ protected T GetSpecificReplyResultValue<T>(ReplyPacket replyPacket)
                 {
                     return JsonConvert.DeserializeObject<T>(replyPacket.resultObject.ToString());
                 }
+                else if (replyPacket.data != null && replyPacket.data.Type == JTokenType.Array)
+                {
+                    return JsonConvert.DeserializeObject<T>(replyPacket.data.ToString());
+                }
```

Comment:
We chatted about this. I ran various unit test and it seems okay.  Basically, if the "data" is an array, your parsing of dataObject below crashes.  The history response is an array.  So, this short circuits the code and parses the raw data directly. The unit test exercised this code successfully.